openapi/components/examples/friends/200-get-friends-count.json

```
@@ -0,0 +1,3 @@
+{
+  "value": 10
```

Shouldn't this be a JSON?
```suggestion
  "value":  { "count": 10 }
```